
Betreff:
[PATCH 1/5] tethering: Add debug print so we know when restarting
Von:
Jukka Rissanen <jukka.rissanen@linux.intel.com>
Datum:
25.06.2013 14:58
An:
connman@connman.net

---
 src/tethering.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/tethering.c b/src/tethering.c
index 223873b..eb05056 100644
--- a/src/tethering.c
+++ b/src/tethering.c
@@ -176,6 +176,7 @@ static void dhcp_server_stop(GDHCPServer *server)
 
 static void tethering_restart(struct connman_ippool *pool, void *user_data)
 {
+	DBG("pool %p", pool);
 	__connman_tethering_set_disabled();
 	__connman_tethering_set_enabled();
 }
--
1.7.11.7

